react/src/pages/Navigation.tsx

```diff
@@ -129,12 +129,16 @@ export interface NavigationProps {
 export default function Navigation({ username, signout }: NavigationProps) {
     const classes = useStyles();
     const [open, setOpen] = React.useState(true);
+    const [pageName, setPageName] = React.useState("Dashboard");
     const handleDrawerOpen = () => {
         setOpen(true);
     };
     const handleDrawerClose = () => {
         setOpen(false);
     };
+    const updatePageName = (name: string) => {
+        setPageName(name)
+    }
```

Comment:
While this function isn't needed because of the reasons I mentioned above, it also really doesn't serve a purpose since it's not doing anything on top of `setPageName`.